### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by d-tak**
Issue 1 - first pass
Thanks Cody!

### CODE REVIEW

**Comment by d-tak** [comment]
https://user-images.githubusercontent.com/99932512/180268525-8561fbd8-7afb-4a35-922e-9e60d22a6e00.mp4

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  height: 40px;
+  border-radius: 4px;
+  border-width: 1px;
+  border-color: #b6b6b6;
+  font-family: "open sans", sans-serif;
+}
+
+/* input:focus {
+  border-color: #562b81;
+} */
```
Comment (codingcodymiller) [inline css/styles.css:55]: Please remove commented code prior to submitting it for review

File: index.html [hunk 2]
```diff
@@ -11,16 +11,65 @@
...
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
   <main>
+    <div class="container header">
+      <h2> Code Journal </h2>
```
Comment (codingcodymiller) [inline index.html:26]: You should put this in a `row` and a `column`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -23,7 +23,9 @@
 <body>
   <main>
     <div class="container header">
-      <h2> Code Journal </h2>
+      <div class="row column-full">
```
Comment (codingcodymiller) [inline index.html:26]: No one element should be both a row and a column. You should give this row a child of a column, and make the `h2` a child of that column.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**